search-app/docs/api.md

```diff
@@ -777,6 +778,7 @@ __Example__
       "id" : "C1200000000-PROV1",
       "original_format" : "ECHO10",
       "browse_flag" : false,
+      "has-variables" : false,
```

Comment:
All the others use underscores ... but the new `has-variables` uses a dash. Do you want underscore here?

The way I understood it (and the way the stubs were provided to EDSC): UMM-JSON has dashes, and regular JSON results have underscores ...

Reply:
Good catch. Done.